ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -46,19 +46,20 @@ private[apiserver] final class StoreBackedCommandExecutor(
       loggingContext: LoggingContext,
   ): Future[Either[ErrorCause, CommandExecutionResult]] = {
     val start = System.nanoTime()
+    val actAsReadAsUnion = commands.actAs ++ commands.readAs
```

Comment:
Can you add a comment that shortly explains why the union is needed for `engine.submit` (and `consume`, for completeness)?

Reply:
Yes, I'll add that 👍

Reply:
I've added a description provided by @rautenrieth-da .